docs/source/advanced_features/fusion_dataset/CADC.rst

```diff
@@ -5,15 +5,15 @@
 This topic describes how to manage the "CADC" dataset.
 
 "CADC" is a fusion dataset with 8 :ref:`sensors<advanced_features/fusion_dataset/fusion_dataset_structure:sensor>` including 7 :class:`cameras<tensorbay.sensor.sensor.Camera>` and 1 :class:`lidar<tensorbay.sensor.sensor.Lidar>`
-, and has :ref:`reference/label_format/Box3D:Box3D` type of labels on the point cloud data.
+, and has :doc:`/reference/label_format/Box3D` type of labels on the point cloud data.
```

Comment:
The word `some` in commit message is not clear.

Reply:
replace ':ref:' directly quoting other rst with ':doc:'？

Reply:
docs: use ":doc:" instead of ":ref:" for specified document link

Reply:
PS. use double quote in commit message